eZ/Publish/API/Repository/SearchService.php

```diff
@@ -18,6 +18,85 @@
 interface SearchService
 {
     /**
+     * Capability flag for scoring feature for use with {@see ::supports()}.
+     *
+     * Scoring, a search feature telling you how well one search hit scores compared to other items in the search result.
+     * When this is supported you can expect search engine to populate SearchHit->score and SearchResult->maxScore
+     * properties as well as sort by this if no sort clauses are specified.
+     */
+    const CAPABILITY_SCORING = 1;
+
+    /**
+     * Capability flag for facets feature for use with {@see ::supports()}.
+     *
+     * Faceted search: https://en.wikipedia.org/wiki/Faceted_search
+     *
+     * Note: Even if search engine tells you this is supported be aware:
+     * - It might not support all facets, by design it will only return facets for the facet builders engine supports.
+     * - Some of the faceting features are still work in process in API and won't be furhter matured before in v2.x
+     *   releases
+     *
+     * @since 6.7.4
```

Comment:
Note to self, misleading `since`. As this, if merged in time for rc, is actually added in `6.12`